bench/load/basic/launchWorkflow.go

```diff
@@ -109,159 +89,161 @@ func launcherWorkflow(ctx workflow.Context, config lib.BasicTestConfig) (string,
-		minSleep := time.Duration(config.MinCadenceSleepInSeconds) * time.Second
-		maxSleep := time.Duration(config.MaxCadenceSleepInSeconds) * time.Second
+	var sleepTime time.Duration
+	if basicTestConfig.MinCadenceSleepInSeconds > 0 && basicTestConfig.MaxCadenceSleepInSeconds > 0 {
+		minSleep := time.Duration(basicTestConfig.MinCadenceSleepInSeconds) * time.Second
+		maxSleep := time.Duration(basicTestConfig.MaxCadenceSleepInSeconds) * time.Second
 		jitter := rand.Float64() * float64(maxSleep-minSleep)
-		sleepTime := minSleep + time.Duration(int64(jitter))
-		input.CadenceSleep = sleepTime
+		sleepTime = minSleep + time.Duration(int64(jitter))
+	}
+	stressWorkflowInput := WorkflowParams{
+		ChainSequence:    basicTestConfig.ChainSequence,
+		ConcurrentCount:  basicTestConfig.ConcurrentCount,
+		PayloadSizeBytes: basicTestConfig.PayloadSizeBytes,
+		PanicWorkflow:    basicTestConfig.PanicStressWorkflow,
+		CadenceSleep:     sleepTime,
 	}
 
-	res := activityResult{}
-	if config.ContextTimeoutInSeconds == 0 {
-		config.ContextTimeoutInSeconds = int(common.DefaultContextTimeout.Seconds())
+	startWFCtxTimeout := common.DefaultContextTimeout
+	if basicTestConfig.ContextTimeoutInSeconds > 0 {
+		startWFCtxTimeout = time.Duration(basicTestConfig.ContextTimeoutInSeconds) * time.Second
 	}
-	for i := params.StartID; i < params.Count; i++ {
-		input.TaskListNumber = rand.Intn(numTaskList)
 
-		workflowOptions.ID = fmt.Sprintf("%s-%d-%d", uuid.New(), params.RoutineID, i)
-		workflowOptions.TaskList = common.GetTaskListName(input.TaskListNumber)
+	for startedID := lastStartedID; startedID < params.Count; startedID++ {
+		stressWorkflowInput.TaskListNumber = rand.Intn(numTaskList)
 
-		startWorkflowContext, cancelF := context.WithTimeout(context.Background(), time.Duration(config.ContextTimeoutInSeconds)*time.Second)
-		we, err := cc.StartWorkflow(startWorkflowContext, workflowOptions, stressWorkflowExecute, input)
+		workflowOptions.ID = fmt.Sprintf("%v-%d-%d", info.WorkflowExecution.ID, params.RoutineID, startedID)
+		workflowOptions.TaskList = common.GetTaskListName(stressWorkflowInput.TaskListNumber)
+
+		startWorkflowContext, cancelF := context.WithTimeout(context.Background(), startWFCtxTimeout)
+		we, err := cc.StartWorkflow(startWorkflowContext, workflowOptions, stressWorkflowExecute, stressWorkflowInput)
 		cancelF()
 		if err == nil {
-			res.IncSuccess()
-			logger.Debug("Created Workflow", zap.String("WorkflowID", we.ID), zap.String("RunID", we.RunID))
+			logger.Debug("Created Workflow successfully", zap.String("WorkflowID", we.ID), zap.String("RunID", we.RunID))
 		} else {
-			res.IncFailed()
-			logger.Error("Failed to start workflow execution", zap.Error(err))
+			if cadence.IsWorkflowExecutionAlreadyStartedError(err) {
+				logger.Debug("Workflow already started in previous activity attempt")
+			} else {
+				logger.Error("Failed to start workflow execution", zap.Error(err))
+				return err
```

Comment:
returning error here may cause the entire load to stop when there's a transient increase in latency?

Reply:
The SDK has done retry already. and we also have retryOption on this activity. 
So here the idea is that the stress workflows can all be started, or the whole bench should just fail. 

The original launchWorkflow would record the num has issues:
* There is no accuracy guarantee. The error could means that workflow has started but server returns some timeout. And then launch may start a new stressWorkflow
* Here I want to simply it. The desired stressWorkflows should always be started using retry. Usually this is not a problem. And then we decide success/failure based on the results of the stressWorkflow(timeout/failure/etc).